junit-platform-launcher/src/main/java/org/junit/platform/launcher/core/ServiceLoaderTestEngineRegistry.java

```diff
@@ -29,8 +29,10 @@
 	public Iterable<TestEngine> loadTestEngines() {
 		Iterable<TestEngine> testEngines = ServiceLoader.load(TestEngine.class,
 			ReflectionUtils.getDefaultClassLoader());
-		LOG.info(() -> "Discovered TestEngines with IDs: "
-				+ stream(testEngines.spliterator(), false).map(TestEngine::getId).collect(toList()));
+		LOG.info(() -> "Discovered TestEngines with IDs: " //
+				+ stream(testEngines.spliterator(), false) //
+						.map(engine -> engine.getId() + "-" + engine.getVersion()) //
```

Comment:
Please build engine information in line with https://github.com/junit-team/junit5/issues/590#issuecomment-265621984